hammer/library/aws/ecs.py

```diff
@@ -0,0 +1,177 @@
+import json
+import logging
+
+from botocore.exceptions import ClientError
+from library.utility import timeit
+from collections import namedtuple
+from library.aws.utility import convert_tags
+
+# structure which describes EC2 instance
+ECSCluster_Details = namedtuple('ECSCluster_Details', [
+    # cluster_id
+    'cluster_arn',
+    # subnet_group_id
+    'cluster_instance_arn'
+])
+
+
+class ECSClusterOperations(object):
+    @classmethod
+    @timeit
+    def get_ecs_instance_security_groups(cls, ec2_client, ecs_client, group_id):
```

Comment:
Where do we use it?

Reply:
This is used for ECS associated insecure SGs,